lib/trailblazer/activity/introspect.rb

```diff
@@ -23,7 +23,12 @@ def find(id = nil, &block)
         end
 
         def collect(strategy: :circuit)
-          @map.keys.each_with_index.collect { |task, i| yield find_with_block { |node| node.task == task }, i }
+          case strategy
```

Comment:
Do we ever use another strategy?

Reply:
What about a first line `return unless strategy == :circuit` instead? Will be much easier to read.

Reply:
> Do we ever use another strategy?
No, but if we have no other strategy in the future. we can remove the param.